bin/appliance_console

```diff
@@ -454,6 +455,12 @@ Static Network Configuration
             press_any_key
             raise MiqSignalError
           end
+        else
+          if !MessageConfiguration.configured?
+            say("\nApplication configuration failed, Messaging configuration is required\n")
+            press_any_key
+            raise MiqSignalError
+          end
```

Comment:
It will throw the error if the user selects `make no messaging changes` while messaging has not already been configured `!MessageConfiguration.configured?`

Reply:
Sounds good, I wonder if we should do this in a few steps though.  First just report if messaging is configured and I think we can merge that at any time.

Starting to require this to be set up is going to have to be merged in sync with a number of other items.

What do you think about dropping this block and simply reporting if messaging is configured or not in appliance console?  Maybe have a "local messaging broker" line similar to the local database one that checks if it is a server specifically.

Reply:
Sounds like a good idea, I'll make the change